graphReconstruct/graphReconstruct_test.go

```diff
@@ -1,5 +1,5 @@
 package graphReconstruct
-
+/*
```

Comment:
Not sure why this is being commented out.  There is another pull request that seems to be fixing it.  Does the fix work (and these comment lines should be removed), or does the proposed fix in the other pull request not fix this and we should comment it out (or fix it)?

Reply:
Yes I think it was fixed by a different pull request. I will update to the one in main.

Reply:
I think the testing file needs to be added to main. That appears to be the source of my compiling error.